components/navbar/Navbar.js

```diff
@@ -172,11 +172,10 @@ export default function Navbar() {
         </div>
 
         <div
-          className={`${
-            isOpen
+          className={`${isOpen
               ? "transform translate-y-0 opacity-100"
               : "transform -translate-y-96 opacity-0 "
-          } md:hidden z-20 absolute t-0 bg-gray-800 transition-all duration-700 ease-in-out  w-full`}
+            } md:hidden z-20 absolute t-0 bg-gray-800 transition-all duration-700 ease-in-out  w-full`}
```

Comment:
if thhis is a formatting change, this should be another PR I think

Reply:
I think the formatter made this changes